cmd/notation/inspect.go

```diff
@@ -247,6 +245,11 @@ func getCertificates(outputFormat string, envContent *signature.EnvelopeContent)
 func printOutput(outputFormat string, ref string, output inspectOutput) error {
 	if outputFormat == cmd.OutputJSON {
 		return ioutil.PrintObjectAsJSON(output)
+	} else {
+		if len(output.Signatures) == 0 {
+			fmt.Printf("%s has no associated signature\n", ref)
```

Comment:
same here. Do we want this extra printout? If no associated signature, the response is just empty.

Reply:
Replied in above comment